03/Product.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+const Product = (props) => {
+    const {name ,isCategory ,add, remove, price, prod} = props;

```

Comment:
Tutaj trzeba to zmienić koncpecję. Zobacz, że przekazujesz wszystkie dane o produkcie, a potem znów przekazujesz cały produkt w `prod` - mamy powielenie danych.